pkgs/by-name/po/poptracker/package.nix

```
@@ -60,9 +63,26 @@ stdenv.mkDerivation (finalAttrs: {
         libsForQt5.kdialog
       ]
     }
+    mkdir -p $out/share/icons/hicolor/{64x64,512x512}/apps
+    ln -s $out/share/${finalAttrs.pname}/icon.png  $out/share/icons/hicolor/64x64/apps/poptracker.png
+    ln -s $out/share/${finalAttrs.pname}/icon512.png  $out/share/icons/hicolor/512x512/apps/poptracker.png
```

```suggestion
    ln -s $out/share/poptracker/icon.png  $out/share/icons/hicolor/64x64/apps/poptracker.png
    ln -s $out/share/poptracker/icon512.png  $out/share/icons/hicolor/512x512/apps/poptracker.png
```
there is a tracking issue about oversuing pname